tests/foreman/rhai/test_rhai.py

```diff
@@ -103,14 +102,14 @@ def test_org_selection_for_rhai(self):
             self.assertIn("Organization Selection Required", result)
 
     @skip_if_not_set('clients')
-    def test_unregister_system_from_rhai(self):
+    def test_positive_unregister_client(self):
```

Comment:
Since this is a `rhai` test module, I thought it would be implied in the test.  But you are right.  I will add `rhai` back.